crates/astria-conductor/src/executor.rs

```diff
@@ -218,34 +212,17 @@ impl<C: ExecutionClient> Executor<C> {
         let prev_block_hash = self.execution_state.clone();
 
         info!(
-            height = block.header.height,
+            height = block.header.height.value(),
             parent_block_hash = hex::encode(&prev_block_hash),
             "executing block with given parent block",
         );
 
-        // parse cosmos sequencer transactions into rollup transactions
-        // by converting them to SequencerMsgs and extracting the `data` field
         let txs = txs
             .iter()
-            .filter_map(|tx| {
-                let body = parse_cosmos_tx(&tx.transaction).ok()?;
-                let msgs: Vec<SequencerMsg> = cosmos_tx_body_to_sequencer_msgs(body).ok()?;
-                if msgs.len() > 1 {
-                    // this should not happen and is a bug in the sequencer relayer
-                    warn!(
-                        msgs = ?msgs,
-                        "ignoring cosmos tx with more than one sequencer message",
-                    );
-                    return None;
-                }
-                let Some(msg) = msgs.first() else {
-                    return None;
-                };
-                Some(msg.data.clone())
-            })
+            .map(|tx| tx.transaction.clone())
```

Comment:
Together with the suggestions above (using the owning iterator we can move out of `tx` and don't need the hot clone):
```suggestion
            .map(|tx| tx.transaction)
```